chap-research.tex

```diff
@@ -905,19 +905,17 @@ \section{Protection Model and Architecture}
 and mappings into multiple underlying ISAs.
 This report describes a software-facing protection model
 (Chapter~\ref{chap:model}) focused on operating systems and compilers,
-specific mapping into the 64-bit MIPS ISA for the purposes of experimentation
-and evaluation (Chapters~\ref{chap:architecture}, ~\ref{chap:cheri-mips}
-and~\ref{chap:isaref-mips}), and architectural sketches for potential integration
-into other ISAs (Chapters~\ref{chap:cheri-riscv} on CHERI-RISC-V,
-Chapter~\ref{chap:cheri-x86-64} on CHERI-x86-64, and Arm
+specific mapping into the 32-bit and 64-bit RISC-V ISA for the purposes of experimentation
+and evaluation (Chapters~\ref{chap:architecture}, ~\ref{chap:cheri-riscv}
+and~\ref{chap:isaref-riscv}), and architectural sketches for potential integration
+into other ISAs (Chapter~\ref{chap:cheri-x86-64} on CHERI-x86-64 and Arm
 Morello~\cite{arm-morello}).
 However, we have taken a ``ground-up'' approach utilizing hardware-software
 co-design to ensure that concrete mapping exist that
 satisfies the practical engineering requirements of architecture,
 microarchitecture, compiler, operating system, and applications.
-At the time CHERI ISAv8 is published, we are in the throes of transitioning
-the bulk of our research from CHERI-MIPS to CHERI-RISC-V and Morello, which
-have learned substantially from our initial experiences implementing CHERI.
+At the time CHERI ISAv9 is published, we have transitioned
+our research from CHERI-MIPS to CHERI-RISC-V and Morello.
```

Comment:
I think we're done so this could be less forward looking?

Reply:
Hmm, I had kept the 'at this time...' clause since it was in the original language, but it could just be dropped.  Not sure if we want some kind of qualification (e.g. "In 2021, we transitioned our research..."

Reply:
Sounds good